modules/3d-tiles/src/lib/parsers/parse-3d-tile-header.ts

```diff
@@ -109,7 +113,10 @@ export function normalizeTileData(
   let tileContentUrl: string | undefined;
   if (tile.content) {
     const contentUri = tile.content.uri || tile.content?.url;
-    tileContentUrl = resolveUri(contentUri, basePath);
+    if (typeof contentUri !== 'undefined') {
```

Comment:
This hasn't quite automerged the change I made on Ib's request, there should be another removal in this file - would you rather I PR separately after this goes in or push a tiny commit on the end of this?

Reply:
oops, sorry! that's my mistake -- I mixed up the two sides of the merge. Should be fixed now! Thanks for catching this! 🙇